buildbot/configure.py

```diff
@@ -181,7 +181,7 @@ def main():
     parser.add_argument("--rocm", action='store_true', help="switch from OpenCL to ROCm")
     parser.add_argument("--rocm-platform", type=str, choices=['AMD', 'NVIDIA'], default='AMD', help="choose ROCm backend")
     parser.add_argument("--arm", action='store_true', help="build ARM support rather than x86")
-    parser.add_argument("--disable-esimd-cpu", action='store_true', help="build without ESIMD_CPU support")
+    parser.add_argument("--enable-esimd-cpu-emulation", action='store_true', help="build with ESIMD_CPU emulation support")
```

Comment:
should it rather be
```suggestion
    parser.add_argument("--enable-esimd-cpu-emulation", action='store_false', help="build with ESIMD_CPU emulation support")
```
?

Reply:
As far as I know, `store_true` means 'True' will be applied to 'enable-esimd-cpu-emulation' member variable with `--enable-esimd-cpu-emuation`. `store_false` means 'False' for the variable. This is same for other plugins that are enabled with similar command line option like CUDA - Line 180.

Reply:
store_true [will ](https://zetcode.com/python/argparse/) set the variable to true when the option is available, otherwise it will remain default (BTW, default=False might be needed). Isn't that the intent?

Reply:
Yes. I verified that current code works as the intent.

Reply:
ok